.github/workflows/cmake-multi-platform.yml

```diff
@@ -0,0 +1,172 @@
+# Github CI config
+# Inspired by https://github.com/libcpr/cpr/blob/master/.github/workflows/ci.yml
+# Runs: clang format check, g++ & clang, release & debug, on Fedora
+
+name: OSTree-TUI CI
+
+on: [push, workflow_dispatch, pull_request]
+
+env:
+  # standard dependencies for building & project
+  BUILD_DEPENDENCIES: "gcc-c++ clang git gcc gdb make cmake automake autoconf"
+  OSTREE_TUI_DEPENDENCIES: "glib2-devel gtk-doc ostree-devel libtool bison liblzf e2fsprogs-devel xz-devel gpgme-devel fuse-devel"
+
+jobs:
+  fedora-clang:
+    strategy:
+      matrix:
+        container: ["fedora:latest"]
+        buildType: [Debug, Release]
+    runs-on: ubuntu-latest
+    container: ${{ matrix.container }}
+    steps:
+    - name: Update package list
+      run: dnf update -y
+    - name: Install Dependencies
+      run: |
+        sudo dnf install -y ${{ env.BUILD_DEPENDENCIES }}
+        sudo dnf install -y ${{ env.OSTREE_TUI_DEPENDENCIES }}
+    - name: Checkout
+      uses: actions/checkout@v3
+    - name: "Build" # & Test"
+      env:
+        CPR_BUILD_TESTS: OFF # ON
+      uses: ashutoshvarma/action-cmake-build@master
+      with:
+        build-dir: ${{ github.workspace }}/build
+        source-dir: ${{ github.workspace }}
+        cc: clang
+        cxx: clang++
+        build-type: Release
+        run-test: false # true
+        ctest-options: ${{ env.CTEST_OPTIONS }}
+
+  fedora-gcc:
+    strategy:
+      matrix:
+        container: ["fedora:latest"]
+        buildType: [Debug, Release]
+    runs-on: ubuntu-latest
+    container: ${{ matrix.container }}
+    steps:
+    - name: Update package list
+      run: dnf update -y
+    - name: Install Dependencies
+      run: |
+        sudo dnf install -y ${{ env.BUILD_DEPENDENCIES }}
+        sudo dnf install -y ${{ env.OSTREE_TUI_DEPENDENCIES }}
+    - name: Checkout
+      uses: actions/checkout@v3
+    - name: "Build" # & Test"
+      env:
+        CPR_BUILD_TESTS: OFF # ON
+      uses: ashutoshvarma/action-cmake-build@master
+      with:
+        build-dir: ${{ github.workspace }}/build
+        source-dir: ${{ github.workspace }}
+        cc: gcc
+        cxx: g++
+        build-type: ${{ matrix.buildType }}
+        run-test: false # true
+        ctest-options: ${{ env.CTEST_OPTIONS }}
```

Comment:
`CTEST_OPTIONS` is not defined? It should be part of your initial `env:` block.

Reply:
There are no tests implemented yet, which is why I kept this empty. To emphasize this, I will comment out these lines. But, as I intend to implement some tests (later / in a different PR), I kept it there for now.